src/utils/keysend.ts

```diff
@@ -13,18 +13,6 @@ export const parseKeysendResponse = (data: KeySendRawData): KeysendResponse => {
   if (data.tag !== TAG_KEYSEND) throw new Error("Invalid keysend params");
   if (data.status !== "OK") throw new Error("Keysend status not OK");
 
-  if (
-    !("customKey" in data.customData[0]) ||
-    data.customData[0]["customKey"] != "696969"
-  )
-    throw new Error("Unable to find customKey");
-
-  if (
-    !("customValue" in data.customData[0]) ||
-    !data.customData[0]["customValue"]
-  )
-    throw new Error("Unable to find customValue");
```

Comment:
I'm not sure that will work, did you test it with a self-custodial lightning address?

https://api.getalby.com/lnurl/lightning-address-details?ln=rolznz@getalby.com 

Further down in this file there is `data.customData[0]["customKey"];` which I think will break since there are no elements in the array (and how do we know it is element 0? we should find the element with `"customKey": "696969"`)

Reply:
afaik it does not need to be 696969 for the podcasting spec for example. The podcasting 2.0 value spec just accepts one customekey and customvalue attribute. 

I've never seen more entries in that array, I'd say [0] is good enough.

but @im-adithya please try it. like this it breaks it seems